src/Compilers/VisualBasic/Portable/DocumentationComments/SourceDocumentationCommentUtils.vb

```diff
@@ -1,14 +1,24 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-Imports System.Text
-Imports System.Threading
 Imports System.Globalization
-Imports Microsoft.CodeAnalysis.Text
-Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
-Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports System.Threading
 
 Namespace Microsoft.CodeAnalysis.VisualBasic
     Friend Module SourceDocumentationCommentUtils
+        Friend Function GetAndCacheDocumentationComment(
+            symbol As Symbol,
+            preferredCulture As CultureInfo,
+            expandIncludes As Boolean,
+            ByRef lazyXmlText As String,
+            cancellationToken As CancellationToken) As String
```

Comment:
>) As String [](start = 50, length = 11)

Consider placing this on a separate line and aligning it with ```Friend```/```End Function```.
```
        Friend Function GetAndCacheDocumentationComment(
            symbol As Symbol,
            preferredCulture As CultureInfo,
            expandIncludes As Boolean,
            ByRef lazyXmlText As String,
            cancellationToken As CancellationToken
        ) As String
             ...
        End Function
```

Reply:
> Consider placing this on a separate line and aligning it with `Friend`/`End Function`.

This is not the natural alignment when <kbd>Enter</kbd> is pressed before `)`. Since this is the first time I've encountered this request in a review, I have updated this instance to follow what you described. However, please consider revising the proposed layout for these scenarios or implementing an improvement to the IDE to eliminate the need to manually adjust indentation.